spec/emites/entities/nfse_spec.rb

```diff
@@ -3,7 +3,124 @@
 describe Emites::Entities::Nfse do
   let(:attributes) do
     {
-
+      "_links" => [
+        {
+            "href" => "https://sandbox.emites.com.br/api/v1/nfse/1",
+            "method" => "GET",
+            "rel" => "self"
+        }, {
+            "href" => "https://sandbox.emites.com.br/api/v1/nfse/1",
+            "method" => "DELETE",
+            "rel" => "destroy"
+        }, {
+            "href" => "https://sandbox.emites.com.br/api/v1/nfse/1/status",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.